examples/api/lib/material/selectable_region/selectable_region.0.dart

```diff
@@ -115,6 +115,9 @@ class _RenderSelectableAdapter extends RenderProxyBox with Selectable, Selection
 
   // Selectable APIs.
 
+  @override
+  List<Rect> get boundingBoxes => <Rect>[paintBounds];
```

Comment:
Is this the best way to set the `boundingBox`? Or should it be based on the `size` `Rect.fromLTWH(0.0, 0.0, size.width, size.height)`?

Reply:
this looks fine